test/specs/collections/Breadcrumb/BreadcrumbSection-test.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import Breadcrumb from 'src/collections/Breadcrumb/Breadcrumb'
```

Comment:
Let's import the actual component here so we don't have a dependency on how it is exposed to the user.

```js
import BreadcrumbSection from 'src/collections/Breadcrumb/BreadcrumbSection'
```